### PR TITLE
{bio}[dummy/] picard v2.9.2 Java-1.8.0_74 (REVIEW)

### PR DESCRIPTION
**Pull request by nathanhaigh**

### CODE REVIEW

File: easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'JAR'
```
Comment (boegel) [inline easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb]: @nathanhaigh There's a custom easyblock for `picard`, why not use that?

If needed, we should update it for more recent `picard` versions?

File: easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  (http://samtools.github.io/hts-specs) format."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/broadinstitute/%(name)s/releases/download/%(version)s/']
+sources = ['%(name)s.jar']
+
+java = 'Java'
+javaver = '1.8.0_74'
+versionsuffix = '-%s-%s' % (java, javaver)
```
Comment (boegel) [inline easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb]: use the `%(javaver)s` template, move `versionsuffix` up right below `version`, cfr. https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/picard/picard-2.1.1-Java-1.8.0_74.eb

File: easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+dependencies = [(java, javaver)]
+
+sanity_check_paths = {
+    'files': sources,
+    'dirs': [],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb]: this is not required when using the `picard` easyblock

File: easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+version = '2.9.2'
+
+homepage = 'http://broadinstitute.github.io/picard/'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM
+  (http://samtools.github.io/hts-specs) format."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/broadinstitute/%(name)s/releases/download/%(version)s/']
+sources = ['%(name)s.jar']
```
Comment (boegel) [inline easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb]: Recent sources aren't versioned anymore?! :(

We should rename the jar to include the version, and add a comment to explain that the renaming should be done manually (until we have proper support to do this automagically in framework).

At the very least, we should include a checksum.

See also https://github.com/hpcugent/easybuild-framework/issues/455#issuecomment-300255441, cc @pneerincx

**Comment by boegel** [comment]
@nathanhaigh ping on remarks?

**Comment by migueldiascosta** [comment]
@nathanhaigh, this PR is being closed for the following reason(s): no activity for > 6 months, obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**